js/components/checkbox.js

```diff
@@ -65,6 +65,24 @@ Fliplet.FormBuilder.field('checkbox', {
     }
   },
   created: function() {
+    var $vm = this;
+
+    if (this.value.length > 0) {
+      var selectedOptions = [];
+
+      this.value.forEach(function(element) {
+        var selectedOption = _.find($vm.options, function(option) {
+          return option.label === element || option.id === element;
```

Comment:
Sometimes an option could have `label` only. Sometimes it could have `label` and `id`. When it has both `label` and `id`, then it **MUST** match the `id`, not the `label`.

If the value is `foo` and the options are:

```js
[
  { label: "foo", id: "bar" },
  { label: "foo2", id: "foo" }
]
```

The correct mat would be `{ label: "foo2", id: "foo" }` because the `id` matches. However, the current implementation has a flaw and will match with `{ label: "foo", id: "bar" }` instead. I recommend something like this:

```suggestion
          return ('label' in option && 'id' in option) ? option.id === value : option.label === value;
```